cadctap/cadctap/tests/test_core.py

```diff
@@ -467,3 +467,14 @@ def test_main(self, schema_mock, query_mock, create_mock, delete_mock,
         main_app()
         calls = [call('tablename', ['path/to/file'], 'tsv')]
         load_mock.assert_has_calls(calls)
+
+        cwd = os.getcwd()
+        if 'tests' in cwd:
```

Comment:
It doesn't work with TESTDATA_DIR/example_query.sql as the other examples?

Reply:
Updated code to use TESTDATA_DIR.